scopes/component/ui/compare/component-compare-context.ts

```diff
@@ -0,0 +1,15 @@
+import { createContext, useContext } from 'react';
+import { ComponentModel } from '@teambit/component';
+
+export type ComponentCompareModel = {
+  base?: ComponentModel;
+  compare: ComponentModel;
+  loading?: boolean;
+  isCompareVersionWorkspace?: boolean;
```

Comment:
should not use "workspace". maybe `compareIsLocalChanges` or `isComparingLocalChanges`?